gensim/models/keyedvectors.py

```diff
@@ -814,17 +948,18 @@ def nbow(document):
         # Compute nBOW representation of documents.
         d1 = nbow(document1)
         d2 = nbow(document2)
-
         # Compute WMD.
         return emd(d1, d2, distance_matrix)
 
-    def most_similar_cosmul(self, positive=None, negative=None, topn=10):
+    def most_similar_cosmul(self, positive=None, negative=None, topn=10, restrict_vocab=None):
         """Find the top-N most similar words, using the multiplicative combination objective,
         proposed by `Omer Levy and Yoav Goldberg "Linguistic Regularities in Sparse and Explicit Word Representations"
         <http://www.aclweb.org/anthology/W14-1618>`_. Positive words still contribute positively towards the similarity,
         negative words negatively, but with less susceptibility to one large distance dominating the calculation.
         In the common analogy-solving case, of two positive and one negative examples,
         this method is equivalent to the "3CosMul" objective (equation (4)) of Levy and Goldberg.
+        restrict_vocab has been added as an optional parameter,
```

Comment:
I don't think it's relevant from the POV of the user that this parameter "has been added". The link to the pull request is helpful, but not ideal. The documentation needs to be self-contained, if there's anything relevant in the PR that belongs in the docs, it should be there directly. Otherwise, the link is unnecessary, in my opinion.